### DISCUSSION TITLE
Setup a Backup SlingBox ?

### DISCUSSION THREAD

**Discussion by ghost** [GerryDazoo/Slinger#71] [General]
So i bought a used SlingBox 350 off eBay for $20 to serve as a backup for my current 350 which i have running Slinger great (thank you so much Gerry).  It powers on and seems to be functional. I was going to connect it to my home network via ethernet and go through a first-time setup. Then i came across this lovely tidbit....

Well...apparently Sling either shutdown their SlingBox "GetStarted" setup page or they don't know there is an issue with it: https://www.slingbox.com/getstarted

This is what i get when i go to that page:

The requested URL was rejected. Please consult with your administrator.

Your support ID is: 13787165963974363701

If anyone has any idea how to configure/setup a SlingBox 350 for initial use without going to that GetStarted page i would love to hear ideas. I have been Googling for the last hour and can't' seem to find any other way to configure a 350. 

If there really is no way to initially configure a SlingBox this close to their November shutdown then I just bought a $20 paper weight lol. Can you open the 350's up easily? I wonder if they have a hard drive or flash memory...and if so... would it be swappable into a "backup" unit of the same model. Just thinking last resort scenarios...

thanks again everyone :)

**Comment by GerryDazoo** [comment]
HAve you tried running SlingDesktop to see if it detects your slingbox?

**Reply by GerryDazoo** [reply]
Is there anyway  to get the password from the vendor?

**Comment by David1946** [comment]
You may be able to use SlingPlayer app on your phone or ipad.

**Comment by ghost** [comment]
Thanks guys! Yeah i tried a bunch of stuff since last night and continued Googling along the way. Probably about 4 hours more of troubleshooting and i been an IT guy for about 20 years now.

When using the SlingPlayer Web player (not desktop app) from my Windows 11 laptop with my existing SlingBox, i was able to get to the little dropdown in the top right hand side of the page to "add new slingbox." But that just takes you to a page that says in order to add a new box you have to download and install the SlingDesktop app. I of course already had that installed and tried using it to detect the new one even more i got to this point. 

But, for good measure, i uninstalled the SlingDesktop app and reinstalled it clean. It does not detect the "new" one i got off eBay. I performed a hard factory reset on the new box per the instructions online and tried the SlingDesktop app again. Even though it does establish a network connection on my home LAN (2nd LED indicator light is on, and i see it show up on my router with an assigned IP) the desktop app still does not detect any other slingboxes aside from my existing one. 

My existing one which of course is tied to my Slingbox account showed up as offline (which is expected since i  had unplugged it and plugged in the new one to avoid conflicts).  I am 99% sure this new box is factory reset and ready to go for an initial setup...however there is just no way to really do that from SlingPlayer's rapidly shrinking online options (via web browser or apps) for configuration. 

I also tried stuff like seeing my android SlingPlayer app on my phone would provide me a way of adding the box, but there does not seem to be a way to do that for the 350s. Sling shows on their website that the newer boxes can be setup wirelessly via your phone but not the 350s and 500s. 

The eBay seller i bought from seems nice but has no idea what the password is on it. I tried clicking that link you provided for finding out what our existing SlingBox admin passwords and critical info is...but it does not work with my new box. It just takes me to the SlingPlayer sign in page where i choose United States and end up right back where i started. I think the only way that link will work is for existing boxes that are tied to your SlingPlayer account. And since i haven't been able to actually perform the "GetStarted" with this new one...i can't exactly add it to my SlingPlayer account.

So basically a circle of madness no matter what i have tried. I hooked my existing working one back up nice and carefully and even got some dust off it. I asked it to give me a few more years lol :) I also plugged it into my APC UPS so that it doesn't go down hard in the future as much when the power goes out lol

I was trying to find pictures of what the 350 looks like if you open it up...i found some and it does not look like there is any type of swappable/clone-able hard drive or flash memory. Everything looks like it is integrated to the tiny board.  Even though i am an IT guy i am not an electrician and would never try sawdering something that tiny with my unsteady hands

Oh well...I guess if something like the power supply goes bad i could potentially just swap the whole board into the new box. Aside from that, like i mentioned before i think i just bought a $20 paperweight. 

I saw there are some other options out there...much more expensive ones that could potentially work should my box ended up dying on me. 

https://max-soft.com/slingbox-diy.htm

$160 - $200+ seems kind of steep though. I don't want or need anything fancy. I am using the RCA connections from my rented Motorola Fios Cable Box into the SlingBox and the cable box. I'm not even using HDMI for the cable box to slingbox connection. That's because the HDMI in my cable box goes to my main TV. So yeah...nothing fancy i still get full HD though and it's always worked great. 

It's definitely an interesting mystery to keep looking into though. If someone finds a way to "configure/reconfigure" a SlingBox bypassing the need to use the SlingPlayer online sites and apps to do so, that would be a game changer and allow folks like us to keep using them for a  long time to come. I mean...if someone hard resets their existing box right now for some reason...they will end up in the same situation that my new "backup" box is in...no way to set the poor little guy back up

**Reply by egadget1** [reply]
I have been able to add six slingbox 350's to my account.  Wonder what the limit is?  I did try and create a new email account to add a couple 500's on it, with no success.  Will try again tonight and let you know.

**Comment by ghost** [comment]
Oh wow! Well, if you were able to add 350s as recently as yesterday/today then maybe something really wrong with my "new" used eBay box. I know for a fact i was able to do the factory reset based on the lights blinking twice after holding down the reset button for about 45 seconds. And after that, it seemed to boot up fine when fully hooked up to my cable box and ethernet using the exact same connections as my existing box (this is when i took the existing one offline and just swapped all the cables into the new one). As mentioned the new one showed up on my router as Slingbox-350 which is the default name. It received a different IP than my offline existing one which also is correct behavior.

So hardware-wise and network connection wise everything with this new box "seems" okay to me. I just cant for the life of me get SlingDesktop app or any of the methods via SlingPlayer in the browser to detect the new one and allow me to initiate that first time configuration. 

It's definitely awesome to hear though that this issue may only just be on my end! I have bad luck so even though i said i was 99% sure this new box was good to go, maybe i was wrong lol. If there are duds and lemons out there i always seem to find them :)

If you are still able to add fresh, factory reset 350s to your SlingPlayer account i would love to hear what tools you used (SlingDesktop app for Windows, SlingPlayer website, SlingPlayer web app, Android app, etc) and steps to get it to detect it and not end up in a circle like me lol. 

Oh and P.S.. It's definitely not an issue with me having too many boxes on my account. I only have one. This used one i got off eBay was supposed to be my second one and serve just as a backup

Thank you again for your responsiveness!

**Reply by GerryDazoo** [reply]
The link I provided many moons ago only will show the boxes that are already on your account.
For grins, startup my server code without specifying on ipaddress in the [SLINGBOX] section. It will do a network probe and hopefully locate your new box. But even so, I haven't been able to get a factory reset box to assign a new password without the Sling server running. I would think there is probably a default password but I have no idea what it might be.

**Reply by GerryDazoo** [reply]
PS. you may want to try turning off your firewall to take that off the table.

**Reply by ghost** [reply]
Thank you sir again for your tips and ideas! I'm gonna mess around with it again this weekend, try this and whatever else I can think of/find :)

It would be awesome if it was something as simple as disabling the Windows firewall on the laptop I was using to try and detect it. I didn't try that though yet cause I figured since the existing slingbox is detected and works from that laptop using SlingPlayer Web, that the ports Sling uses must already be open. But it's definitely worth a shot 

Thanks again everyone!

**Comment by egadget1** [comment]
Not sure if this is why its not letting me add more slingboxes to my account.   I was using my old win7 laptop and it was adding ok.  Then last night it updated java.  Now I can't add anymore.  I think we might have to use an old version of Java for it to work?

**Comment by ghost** [comment]
thanks everybody for your ideas and tips. I messed with the new 350 (the one i wanted to make a backup that i just got off eBay last week) some more this weekend. 

Well, i believe that i bricked it via another factory/hard reset attempt. So yeah it's definitely a $20 paperweight now. Oh well. It was worth a shot. My guess is that even though it was (prior to me bricking it) seeming to power up correctly and establish an IP on my router, I think it was probably messed up from the get-go. That's likely why it would not detect when i tried for hours to go through the initial first time setup. 

There are about 5-7 more Slingbox 350s on eBay right now that are being sold for $19-25. I'm thinking about trying this again...but obviously with my luck I'm a little leery. 

If anyone is able to setup a Slingbox 350 that has been factory reset let me know. As in, they are able to get it to be detected via something like the SlingDesktop app during the initial config process, add it successfully to their SlingBox account, etc. I'd love to know that someone was able to get it working despite Sling taking down a lot of the resources we had been using to set these devices up such as that GetStarted page i cited at the very beginning of this post

Hope everyone has a great rest of their weekends :)

**Reply by GerryDazoo** [reply]
In another thread someone suggested the default password after a factory reset was "admin". Run my code with that password and see if it connects. There may be hope....

**Comment by ghost** [comment]
Thank you sir! I would love to try that but over the weekend i seem to have bricked the new "used" backup 350 i got off ebay . It does nothing but flash lights now, reboot itself, rinse repeat. All i did was try another factory/hard reset on it Saturday and it ended up in this state.

So as mentioned in my last reply, I'm leery to go buy another backup Slingbox because of my luck. But, I definitely want to get another one working before they shut down their servers. I expanded my eBay search beyond just the 350s and found an M2 for only $16

https://www.ebay.com/itm/255736576530

It doesn't come with the power supply...but that's okay because the power supply on the 350 i just bricked works fine and should work on an M2 from what i been able to tell anyway. Maybe id have better luck trying to get an M2 up and running

**Reply by egadget1** [reply]
Hi,  I will give the admin slingbox 350 a try tonight.  I did just setup a SB300 with any issue.  I would just get a Slingbox 500 for backup those seem to update and configure fine via the onscreen menu.

**Reply by egadget1** [reply]
Well I factory reset a slingbox 350 (ie I held down the reset swicth for 10 sec).  Connected to my tivo mini, edited the config.ini file where I replaced the admin password with "admin", your software detected it as "My Slingbox" and I was able to stream it!

**Reply by GerryDazoo** [reply]
Great news. Thanks!

**Comment by ghost** [comment]
okay since everyone seems to be having good luck...I think i'm going to give it another shot. I really want to get a backup slingbox up and running on Gerry's awesome Slinger just for piece of mind because eventually my working original 350 will die. 

I was ready to get that M2 as a backup off eBay (doesn't have a power supply). I figure that my 350 power supply will work on it but I'm not certain. The 350 is 5V 3A max and the M2 is 5V 2A max. So in theory, it should since the 350 power supply actually can put out more maximum amps than the M2 needs. 

Thoughts?

If I'm right I'll get that one. 

If I'm wrong then ill find a "backup" slingbox that comes with a charger. I saw another 350 on ebay going for $21 and an M2 with a charger for $25

I like the suggestion on looking for a used slingbox 500 since you can do the onscreen setup. Those are running about $10 more expensive. i saw one with the power adapter and remote for like $34. Since i was already burned once with that 350 that bricked...I'm trying to stick to something under $25 lol.

**Comment by ghost** [comment]
Well, i got the "backup" Slingbox 350 from eBay UN-bricked :) So i don't have to get another one now after all! All i did was give it some solid love taps on the bottom and then i tried yet one last hail mary hard reset and this time it worked - no more crazy lights flashing. 

Thank you for the awesome tips in this post! I decided to bypass trying to do the first-time setup using SlingBox's resources such as the SlingDesktop app. Instead i just made a copy of Gerry's config.ini file specifically for this backup box, edited it accordingly, and fired up the server on my PC. It found the backup Slingbox without issue and i was able to stream perfectly from every device and TV just like my main Slingbox. 

So yes, the default password is most definitely "admin" after doing a hard factory reset on a SlingBox 350 and there is no need to even mess with Sling's messed up and shrinking options they have for setting up a box for initial use. Just bypass that and use Slinger straight from the jump :)

The ONLY issue with not going through a first-time setup via Sling's resources, the Slingbox 350's logo LED light (the in-use light) will never stop flashing. That flash indicates it's been reset and is awaiting that first time setup that i never did and don't plan to do. Of course when you are streaming from it the light goes solid per normal. But yeah when you aren't streaming if the flashing gets annoying just put a tiny piece of black electrical tape on it. 

Again, thank you to Gerry and also to everyone else who gave feedback on this!

**Reply by GerryDazoo** [reply]
Just wondering about the Remote Control configuration. Is there a built-in default remote? Can you set your VideoSource in your config.ini and report what keycodes (if any) are listed? Not having a source/remote configured is probably why it's flashing

**Comment by ghost** [comment]
Not sure, i haven't looked into any of the remote settings. I have a chance this weekend, ill hook the backup 350 backup and see if i can see what codes are listed. Thanks yet again for another tip! That could definitely explain the flashing. I just figured the flashing was due to it still being in some kind of first-time config mode awaiting a connection to their servers via SlingDesktop app or something

Since i have Verizon Fios i actually have always just used the Fios mobile app on my phone as the remote for my cable box whenever i want to change the channel and i am using my Slingbox on another TV. 

By the way, being able to stream on more than 1 device with your Slinger is absolutely a game changer! So cool! I had the ESPN Monday Night Football game on 7 devices at once last week just to see how it performed and my CPU on my main Windows 11 PC that runs Slinger didn't even go up more than 1%

**Comment by floormanager1** [comment]
Hello, I've had a similar event.  After success with Gerry's server on a 350, I hot-footed over to ebay and bought a new M1 for a few £s.  Unboxed it and tried to set it up using Slingbox servers and could not get it to work as I did not know the admin password.  During the frustrations, somehow in the process, I managed to lose my 350 from my account (can't get that back either).  Anyway, I decided to do a hard reset, pointed my config.ini at it (not knowing the finderID either) and it is streaming quite happily.  The remote is not working  yet, although it does on the 350, might have to look at that.  Is there anyway of getting the finders ID or is it unnecessary now?
I am so pleased to be able to use Slingboxes after 9th Nov!  We owe Gerry a lot of beers!

**Reply by GerryDazoo** [reply]
Thanks for the virtual beers. The reason the remote doesn't work is because you haven't configured a remote for your video source and the slingbox doesn't know what IR codes to send. you "should" still be able to set that up with the slingbox desktop software for now. After the 9th you will need to load a third party remote definitions file.
http://www.hifi-remote.com/forums/dload.php?action=category&cat_id=87

**Reply by floormanager1** [reply]
I did nearly have it set but did not know the admin password at the time.  Like I said, I even managed to lose the 350 Slingbox from my account and I do not seem to be able to get that back.  I'm not too concerned about that as my Mum was the only one to use that via the Slingbox Firestick app, which will be dead in a month's time.  I've set her up with MX player now but she's struggling to work out how to do it.  Luckily, I found a remote way of connecting to the Firestick via Vysor, so I can fire it up for her. All the best, Paul.

**Reply by floormanager1** [reply]
Strangely now, after a re-boot, I've managed to get the Slingbox Desktop to connect the M1!  Now, do I try and try and get the 350 back onto my account?

**Reply by sp222222** [reply]
yeah, floormanager1 - i'm in USA so not sure if other countries are having issues adding slingboxes. I bought a 500 about 3 weeks ago and added it using the desktop app that windows uses. you MUST hold that reset button down until lights flash to get it to clear out all other setups. I thought my ebay sender did that but they did not. after that I was able to add the 500 to my network (was not happy I could not change the default port from 5201 thought) I added to my list of slingboxes (six of them) and I'm up and running with slinger.

**Comment by Pthepeg** [comment]
I picked up a used Slingbox350, did a hard reset and can confirm the password  "admin" did work with slinger.
I used SlingplayerDesktop on Windows10 and was able to find the new 350 and configure the remote for my settop box.
And then found the long password using the link in the slinger readme notes.
Thanks to all the previous comments and help.

**Reply by bradut** [reply]
Had a similar issue trying to add a new [really sealed] HD-PRO from ebay.
The setup and firmware update only worked in Windows 7 (using Sling player 2.0.4.552).
In Windows 10 they both got stuck at a certain point.
